### PR TITLE
chore(turbo-ignore): update release flow

### PR DESCRIPTION
**Pull request by tknickman**
Pull turbo-ignore into the standard release flow

### CODE REVIEW

File: cli/Makefile [hunk 1]
```diff
@@ -162,52 +161,6 @@ stage-release: cmd/turbo/version.go
 	git tag "v$(TURBO_VERSION)"
 	git push origin staging-$(TURBO_VERSION) --tags --force
 
-.PHONY: publish
-publish: clean build
```
Comment (tknickman) [inline cli/Makefile:166]: This is no longer used - it's replaced by `publish-turbo`

File: cli/Makefile [hunk 2]
```diff
@@ -225,12 +178,14 @@ publish-turbo: clean build
 	cd $(CLI_DIR)/../packages/turbo && pnpm pack --pack-destination=$(CLI_DIR)/../
 	cd $(CLI_DIR)/../packages/create-turbo && pnpm pack --pack-destination=$(CLI_DIR)/../
 	cd $(CLI_DIR)/../packages/turbo-codemod && pnpm pack --pack-destination=$(CLI_DIR)/../
+	cd $(CLI_DIR)/../packages/turbo-ignore && pnpm pack --pack-destination=$(CLI_DIR)/../
```
Comment (mehulkar) [inline cli/Makefile:181]: 😍

File: packages/turbo-ignore/tsup.config.ts [hunk 3]
```diff
@@ -3,7 +3,7 @@ import { defineConfig, Options } from "tsup";
 export default defineConfig((options: Options) => ({
   entry: ["src/index.ts"],
   format: ["cjs"],
-  minify: true,
+  minify: false,
```
Comment (mehulkar) [inline packages/turbo-ignore/tsup.config.ts:6]: why change this? wouldn't a minified build download faster from npm?

**Review by mehulkar — APPROVED** [review]
One comment, but otherwise 👍🏾 !

File: packages/turbo-ignore/tsup.config.ts (hunk 3 above)
Comment (tknickman) [reply packages/turbo-ignore/tsup.config.ts:6]: Yes! Leftover from a bundling test I was doing. Thanks for catching 🙇🏼

**Pull request merged**